booking/src/main/java/ru/org.dsr/controller/ResultController.java

```diff
@@ -0,0 +1,77 @@
+package ru.org.dsr.controller;
+
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+import ru.org.dsr.domain.Item;
+import ru.org.dsr.domain.ItemID;
+import ru.org.dsr.domain.Comment;
+import ru.org.dsr.exception.RobotException;
+import ru.org.dsr.search.SearchKinopoiskJSOUP;
+import ru.org.dsr.services.CommentsService;
+import ru.org.dsr.services.ManagerSearch;
+
+import java.util.List;
+
+@RestController
+public class ResultController {
+
+    private static final Logger log = Logger.getLogger(SearchKinopoiskJSOUP.class);
+
+
+    @Autowired
+    CommentsService commentsService;
+    @Autowired
+    ManagerSearch manager;
+
+    private Item item;
+
+    @ResponseBody
+    @RequestMapping(value = "/result-clear", method = RequestMethod.DELETE)
+    public boolean clearBD() {
+        commentsService.clear();
+        item = null;
+        log.info("DB clear");
+        return true;
+    }
+
+    @GetMapping("/result")
+    public ModelAndView toResult() {
+        ModelAndView mav = new ModelAndView();
+        mav.setViewName("result");
+        return mav;
+    }
+
+    @ResponseBody
+    @RequestMapping(value = "/result-getComments", method = RequestMethod.POST)
+    public List<Comment> getComments(@RequestBody int page) {
```

Comment:
Shoulb be GET, not POST. Please, take a look at REST API guidelines. 
Page could be passed via query parameters.

Reply:
Generally, I believe, the API should've contained only one `POST /search` endpoint, with parameters for searching book and paging in body, which will return search results in `json` format. 
Or it could be done via `GET /comments` and parameters passed via query string.